internal/v2/models/access_flow_to_request.go

```diff
@@ -161,13 +161,15 @@ func convertConditionToUpsertRequest(ctx context.Context, model AccessFlowCondit
 
 	condition.MatchOperator.SetTo(model.MatchOperator.ValueString())
 
-	var values []string
+	if !model.Values.IsNull() {
```

Comment:
api doesn't except nil value...
so I had to do the "if" here.

can be removed if fixed in the API.